### ISSUE TITLE
Yank all 0.1 and 0.2 releases

### ISSUE THREAD

**Issue by xfix** [rust-lang/log#480]
All log `0.1.x` and `0.2.x` releases don't compile on post-1.0.0 Rust due to `std::rt` dependency. Doing so would help `cargo update -Z minimal-versions` builds if those versions would be all yanked.

**Comment by KodrAus** [comment]
Hi @xfix :wave:

Hmm, I'm trying to catch up on the context around this. It looks like the goal is to try patch around a lack of usable minimal version bounds in dependent crates.

I'm concerned about sending the wrong kind of signal here. I wouldn't expect a crate to be yanked unless there was a critical reason to do so, like a CVE disclosure. Authors could normally just decide how to manage their crates however they want, but since this project lives in the Rust organization I think we do need to be especially considerate.

I personally don't believe this is something we should do, even though those versions are mostly unusable, but would be interested to see what @rust-lang/libs thinks. You have inspired me to go add minimal version checks to our CI here in `log` though.

**Comment by m-ou-se** [comment]
I agree with @KodrAus, but I don't have a strong opinion on this either way.
